_aggregations/pipeline/index.md

```diff
@@ -0,0 +1,202 @@
+
+Pipeline aggregations are executed after all other sibling aggregations. This has performance implications. For example, using the `bucket_selector` pipeline aggregation to narrow a list of buckets does not reduce the number of computations performed on omitted buckets.
+{: .note}
+
+Pipeline aggregations cannot be sub-aggregated, but can be chained with other pipeline aggregations. For example, you calculate a second derivative by chaining two consecutive `derivative` aggregations. Keep in mind that pipeline aggregations append to existing output. For example, computing a second derivative by chaining `derivative` aggregations outputs both the first and second derivatives.
+
+## Pipeline aggregation types
+
+Pipeline aggregations are of two types, [sibling](#sibling-aggregations) and [parent](#parent-aggregations).
+
+### Sibling aggregations
+
+A _sibling_ pipeline aggregation takes the output of a nested aggregation and produces new buckets or new aggregations at the same level as the nested buckets.
+
+A sibling aggregation must be a multi-bucket aggregation (have multiple grouped values for a certain field), and the metric must be a numeric value.
+
+`min_bucket`, `max_bucket`, `sum_bucket`, and `avg_bucket` are common sibling aggregations.
+
+### Parent aggregations
+
+A _parent_ aggregation takes the output of an outer aggregation and produces new buckets or new aggregations at the same level as the existing buckets.
+
+The specified metric for a parent aggregation must be a numeric value.
+
+We strongly recommend setting `min_doc_count` to `0` (the default for `histogram` aggregations) for parent aggregations. If `min_doc_count` is greater than `0`, then the aggregation omits buckets, which might lead to incorrect results.
+{: .important}
+
+`derivatives` and `cumulative_sum` are common parent aggregations.
+
+## Buckets path
+
+A pipeline aggregation uses the `buckets_path` parameter to access the results of other aggregations.
+The `buckets_path` parameter has the following syntax:
+
+```
+buckets_path = <AGG_NAME>[<AGG_SEPARATOR>,<AGG_NAME>]*[<METRIC_SEPARATOR>, <METRIC>];
+```
+
+| Element | Literal character | Description |
+| :-- | :-- | :-- | 
+| `AGG_NAME` |  | The name of the aggregation. |
+| `AGG_SEPARATOR` | `>` |  The character used to separate aggregation names. |
+| `METRIC_SEPARATOR`| `.` |  The character used to separate final aggregation from its metrics. |
+| `METRIC` |  | The name of the metric. Required for multi-value metric aggregations. |
+
+For example, `my_sum.sum` selects the `sum` metric of an aggregation called `my_sum`. `popular_tags>my_sum.sum` nests `my_sum.sum` into the `popular_tags` aggregation.
+
+# Data gaps
```

Comment:
We should move this down because buckets path example should be under buckets path.